### DISCUSSION TITLE
growup follower

### DISCUSSION THREAD

**Discussion by MeMAURON** [community/community#135079] [New to GitHub] [answered]
Hello there How can I increase my followers and improve?

**Comment by UjjwalSaini07** [comment, marked as answer]
in github u increase ur follower by:
1. following more people and by the side they also follow u  **_-- Very easy_**
2. make good projects then anyone like ur project so they follow u **_-- but this is a hard and time taking process_**
3. Make increase ur reach to getting more follower and become star celebrity as a developer  **_- Medium_**

**Reply by affen88plus** [reply]
1

**Comment by Jayantnis** [comment]
@affen88plus right 💯

**Reply by Jayantnis** [reply]
response #2
